pkg/vm/memory/memory_value.go

```diff
@@ -315,3 +315,19 @@ func (mv *MemoryValue) Uint64() (uint64, error) {
 func (mv *MemoryValue) addrUnsafe() *MemoryAddress {
 	return (*MemoryAddress)(unsafe.Pointer(&mv.felt))
 }
+
+func (memory *Memory) GetConsecutiveValues(addr MemoryAddress, size int16) ([]MemoryValue, error) {
```

Comment:
what about renaming `GetConsecutiveValues` : `GetConsecutiveMemoryValues` ?

Reply:
as we use in this file `MemoryValueFromMemoryAddress` , `MemoryValueFromFieldElement`, etc..

Reply:
Done 👍🏽